sdl_ios/SmartDeviceLink/SDLIAPTransport.m

```diff
@@ -295,16 +311,15 @@ - (void)openSession {
         } else {
             if ([self.protocolString isEqualToString:CONTROL_PROTOCOL_STRING]) {
```

Comment:
Sure, the lines

```objc
if ([self.protocolString isEqualToString:CONTROL_PROTOCOL_STRING]) {
                [SDLDebugTool logInfo:@"Session Not Opened (Control Protocol)" withType:SDLDebugType_Transport_iAP toOutput:SDLDebugOutput_All toGroup:self.debugConsoleGroupName];
            } else {
                [SDLDebugTool logInfo:@"Session Not Opened" withType:SDLDebugType_Transport_iAP toOutput:SDLDebugOutput_All toGroup:self.debugConsoleGroupName];
            }
```

used to contain the connection retry. Now they only log. If the logs are still necessary, then they can remain, but if not, then they could be removed since no active code is being run within them.